### PR TITLE
docs: update

### PR DESCRIPTION
**Pull request by Nixie404**
This PR fixes spelling in the Install UN1CA (Custom recovery) section.

### CODE REVIEW

File: docs/guide/recovery.md [hunk 1]
```diff
@@ -20,12 +20,12 @@ nav_order: 2
   <img loading="lazy" src="/assets/images/twrp.png" width="30%"/>
 </p>
 
-Installing custom ROM's via recovery has been the standard since Android early times, and today it's still the way to go in most devices.
+Installing custom ROMs via a recovery has been the standard since Android early times, and today it's still the way to go in most devices.
 This guide will not go in deep details about how to install a custom recovery, as the procedure might differ from device to device, so please refer to the XDA community page of [your device]({% link devices/index.md %}).
 
-To install UN1CA, a custom recovery and/or other unofficial images, your device's bootloader **must be unlocked**, if you're unsure please give a look at the ["Unlocking the bootloader"]({% link guide/bootloader.md %}) page.
+To install UN1CA, a custom recovery and/or other unofficial images are required and your device's bootloader **must be unlocked**. If you are unsure please take a look at the ["Unlocking the bootloader"]({% link guide/bootloader.md %}) page.
```
Comment (salvogiangri) [inline docs/guide/recovery.md]: Remove "are required and" as that's not what I meant

Comment (salvogiangri) [reply docs/guide/recovery.md]: This also must match the Download mode guide:
https://github.com/BlackMesa123/UN1CA/blob/073aed5b12625bd6e5550f4f75a107fd19423fe2/docs/guide/download.md?plain=1#L30

File: docs/guide/recovery.md [hunk 2]
```diff
@@ -37,11 +37,11 @@ Once you've made sure your device's bootloader is unlocked and have installed a
...
 - Open the Install menu, select the UN1CA package you previously downloaded and flash it. UI might vary depending which recovery you've installed.
 - Your specific device installer might have a "Post-install" feature enabled which will automatically reboot your device a few times. Don't worry about it.
-- Once the install has finished, open the Wipe → Format Data menu, type "yes" and confirm. **This step is only required the very first time you install the ROM.**
+- Once the installation has finished, open the Wipe → Format Data menu, type "yes" and confirm. **This step is only required the very first time you install the ROM.**
 
-If you have followed every step correctly, the ROM has been installed succesfully and you can now exit recovery by selecting Reboot → System.
+If you followed every step correctly, the ROM has been installed succesfully and you can now exit recovery by selecting Reboot → System.
 
-If any other error/issue occurs, please refer to the [Troubleshooting]({% link faq/index.md %}) section.
+If you encounter any error/issue, please refer to the [Troubleshooting]({% link faq/index.md %}) section.
```
Comment (salvogiangri) [inline docs/guide/recovery.md:44]: This must match the Download mode guide:
https://github.com/BlackMesa123/UN1CA/blob/073aed5b12625bd6e5550f4f75a107fd19423fe2/docs/guide/download.md?plain=1#L65

**Review by salvogiangri — CHANGES_REQUESTED** [review]

File: docs/guide/recovery.md (hunk 2 above)
Comment (Nixie404) [reply docs/guide/recovery.md:44]: In this PR I also updated the download page

File: docs/guide/recovery.md (hunk 1 above)
Comment (Nixie404) [reply docs/guide/recovery.md]: Fixed.

**Review by salvogiangri — APPROVED** [review]

**Pull request merged**